rust/rsc/src/rsc/main.rs

```diff
@@ -102,7 +104,16 @@ fn create_router(conn: Arc<DatabaseConnection>, config: Arc<config::RSCConfig>)
 }
 
 async fn create_standalone_db() -> Result<DatabaseConnection, sea_orm::DbErr> {
-    let db = Database::connect("sqlite::memory:").await?;
+    let shim_db = Database::connect("postgres://127.0.0.1/shim").await?;
+    let mut buf = [0u8; 24];
+    OsRng.fill_bytes(&mut buf);
+    let rng_str = HEXLOWER.encode(&buf);
+    let db = format!("db_{}", rng_str);
+    shim_db
+        .execute_unprepared(&format!("CREATE DATABASE {}", db))
+        .await?;
+    drop(shim_db);
```

Comment:
Can you explain what `drop` is doing here/why it is needed?

Reply:
I don't think its needed but it causes the shim_db to be droped before we create a new one so that we don't have two connections at once.

It also might help enforce that the follow up connection uses the same view version of the databases as the one we just created (e.g. drop might wait for transactions to be committed but I'm not too sure about that and it shouldn't be needed strictly here, I think the await is enough to know that the commit has happened and any new connections will see the new database.